packages/@react-aria/tag/docs/useTagGroup.mdx

```diff
@@ -0,0 +1,296 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/tag';
+import typesDocs from 'docs:@react-types/shared/src/events.d.ts';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+import packageData from '@react-aria/tag/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import tailwindExample from 'url:./tailwind.png';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import Anatomy from './anatomy.svg';
+
+---
+category: Collections
+keywords: [tag, aria]
+---
+
+# useTagGroup
+
+<PageDescription>{docs.exports.useTagGroup.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useTagGroup']} />
+
+## API
+
+<FunctionAPI function={docs.exports.useTagGroup} links={docs.links} />
+<FunctionAPI function={docs.exports.useTag} links={docs.links} />
+
+## Features
+
+* Exposed to assistive technology as a grid using ARIA
+* Keyboard navigation support including arrow keys, home/end, page up/down, and delete
+* Keyboard focus management and cross browser normalization
+* Labeling support for accessibility
+* Support for mouse, touch, and keyboard interactions
+
+## Anatomy
+
+<Anatomy />
+
```

Comment:
Maybe we should also include the tag group label in this diagram? Might be confusing since there is something called "Label" there but it's the individual tag label rather than the one for the whole tag group. Maybe make that more specific too.